### PR TITLE
Expose env var for demo dynamic plugin cypress testing

### PR DESCRIPTION
**Pull request by jcaianirh**
For: https://issues.redhat.com/browse/CONSOLE-2925

### CODE REVIEW

File: ci-operator/config/openshift/console/openshift-console-master.yaml [hunk 1]
```diff
@@ -9,6 +9,9 @@ images:
 - dockerfile_path: Dockerfile
   from: base
   to: console
+- dockerfile_path: Dockerfile.plugins.demo
+  from: base
+  to: plugin-pull-spec
```
Comment (spadgett) [inline ci-operator/config/openshift/console/openshift-console-master.yaml:14]: I'd call this something like `console-demo-plugin`

Comment (jcaianirh) [reply ci-operator/config/openshift/console/openshift-console-master.yaml]: @spadgett ok thanks, done

**Review by jhadvig — CHANGES_REQUESTED** [review]
The change looks good. Please squash the commits and lets merge this baby ! 😄

**Review by jhadvig — APPROVED** [review]
/lgtm
/approve

**Comment by bparees** [comment]
image name is approved

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**